src/toolkits/object_detection/object_detector.cpp

```diff
@@ -341,28 +341,8 @@ void object_detector::import_from_custom_model(variant_map_type model_data,
   if (model_iter == model_data.end()) {
     log_and_throw("The loaded turicreate model must contain '_model'!\n");
   }
-  const flex_dict& model = variant_get_value<flex_dict>(model_iter->second);
-  auto shape_iter = model_data.find("_grid_shape");
-  size_t height, width;
-  if (shape_iter == model_data.end()) {
-    height = 13;
-    width = 13;
-  } else {
-    std::vector<size_t> shape =
-        variant_get_value<std::vector<size_t>>(shape_iter->second);
-    height = shape[0];
-    width = shape[1];
-  }
-  model_data.emplace("grid_height", height);
-  model_data.emplace("grid_width", width);
-
-  model_data.emplace("annotation_scale", "pixel");
-  model_data.emplace("annotation_origin", "top_left");
-  model_data.emplace("annotation_position", "center");
-
-  state.clear();
-  state.insert(model_data.begin(), model_data.end());
 
+  const flex_dict& model = variant_get_value<flex_dict>(model_iter->second);
```

Comment:
reorganize the code to follow the guide of C++: when you use it, you declare it.